src/tuning.ts

```diff
@@ -5,9 +5,14 @@
 
 import { Vex } from './vex';
 import { Flow } from './tables';
-
 export class Tuning {
-  static get names() {
+  private numStrings: number;
+
+  private tuningString: string;
+
+  private tuningValues: number[];
```

Comment:
can we please use `protected` over `private` so that developers can subclass Tuning to get at these attributes?

Reply:
agree

Reply:
@mscuthbert @0xfe both `private` and `protected` change the interface anyway. Should  we leave them `public`?

Reply:
My preference is as @mscuthbert to make them protected. If we go this way, I would also: 
- delete `tuningStrings`which is not used
- delete `numStrings` whose content is not obvious and use `tuningValues.length` instead

Reply:
I think `protected` is good. I'd prefer not to change any implementation details in this PR.

Reply:
@0xfe ok, how should I proceed with the other topics. One issue collecting all the issues found in the migration or one issue per topic?

![image](https://user-images.githubusercontent.com/22865285/113049360-d722a200-91a3-11eb-8537-5e321d263bf7.png)

Reply:
ok